bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/handler/DaikinMadokaHandler.java

```diff
@@ -690,6 +786,36 @@ public void receivedResponse(SetPowerstateCommand command) {
         }
     }
 
+    @Override
+    public void receivedResponse(GetOperationHoursCommand command) {
+        logger.debug("receivedResponse(GetOperationHoursCommand command)");
+
+        DecimalType indoorPowerHours = command.getIndoorPowerHours();
+        DecimalType indoorOperationHours = command.getIndoorOperationHours();
+        DecimalType indoorFanHours = command.getIndoorFanHours();
```

Comment:
These should be QuantityTypes now instead.

Reply:
Fixed. Have also fixed other units (SetPoint, IndoorTemperature, OutdoorTemperature...) all migrated to QuantityType